strongbox-commons/src/test/java/org/carlspring/strongbox/artifact/coordinates/NugetPackageConverterTest.java

```diff
@@ -8,24 +8,25 @@
 {
 
     @Test
-    public void testArtifactPathToCoordinatesConversion() throws Exception
+    public void testArtifactPathToCoordinatesConversion()
+                                                          throws Exception
```

Comment:
Cosmetic/trivial:
The code style across the project is:

    public void testArtifactPathToCoordinatesConversion()
                throws Exception